Source/FieldSolver/ElectrostaticSolver.cpp

```diff
@@ -165,15 +165,18 @@ WarpX::AddSpaceChargeField (WarpXParticleContainer& pc)
         BoxArray nba = boxArray(lev);
         nba.surroundingNodes();
         rho[lev] = std::make_unique<MultiFab>(nba, DistributionMap(lev), 1, ng);
+        rho[lev]->setVal(0.);
```

Comment:
@RemiLehe @dpgrote did we leave this non-zero intentionally (so we can add to `rho` multiple times)?

Reply:
oh wait, looks like it's a temporary here.

Reply:
Yeah, it's a temporary. I added this call because I can no longer rely on a call below with `reset=true`